data-sources/jfrog-platform/config.yml

```diff
@@ -1,8 +1,8 @@
-id: jfrog-platform
-displayName: JFrog Platform
-description: Observe JFrog Platform and gain insights using New Relic Observability platform
+id: jfrog-platform-cloud
```

Comment:
Since your quickstart will generate a new data source named `jfrog-artifactory-cloud`, there's no requirement to alter the existing data source [jfrog-platform](https://newrelic.com/instant-observability/jfrog-platform). Undertaking this action would simply lead both the new and existing data sources to the same page, rendering the modification unnecessary.

Reply:
That was a mix-up of two config files, I'm fixing that. Thanks!